README.md

```diff
@@ -10,15 +10,52 @@
 - [Documentation](#Documentation)
 - [Links](#Links)
 
-## <a name='maintenance'>Library maintenance</a>
+## Library maintenance
 
-This client library is supported but in maintenance mode only. We are fixing necessary bugs and adding essential features to ensure this library continues to meet your needs for accessing Google APIs. Non-critical issues will be closed. Any issue may be reopened if it is causing ongoing problems.
+This client library is supported but in maintenance mode only. We are fixing necessary bugs and
+adding essential features to ensure this library continues to meet your needs for accessing Google
+APIs. Non-critical issues will be closed. Any issue may be reopened if it is causing ongoing problems.
 
-If you're working with **Google Cloud Platform** APIs such as Datastore, Pub/Sub and many others, consider using the [Cloud Client Libraries for Java](https://github.com/GoogleCloudPlatform/google-cloud-java) instead. These are the new and idiomatic Java libraries targeted specifically at Google Cloud Platform Services.
+If you're working with **Google Cloud Platform** APIs such as Datastore, Pub/Sub and many others,
+consider using the [Cloud Client Libraries for Java](https://github.com/GoogleCloudPlatform/google-cloud-java)
+instead. These are the new and idiomatic Java libraries targeted specifically at Google Cloud
+Platform Services.
 
-## <a name='Overview'>Overview<a/>
+## Building locally
 
-The Google APIs Client Library for Java is a flexible, efficient, and powerful Java client library for accessing any HTTP-based API on the web, not just Google APIs. 
+##### One time setup
+
+```
+mkdir /tmp/foo && cd /tmp/foo
+wget --no-check-certificate https://dl.google.com/dl/android/maven2/com/google/android/gms/play-services-basement/8.3.0/play-services-basement-8.3.0.aar
```

Comment:
Why `--no-check-certificate`?

Reply:
*shrug* Copy pasted what I had locally. I can remove, it's probably more canonical to let users figure that out themselves?

Reply:
Why do you need that locally? That means the TLS certs are broken, and isn't something that should generally be ignored.

Reply:
Removed the line + rebasing into master while we continue the conv offline